cron/cron.go

```diff
@@ -48,7 +51,7 @@ type JobConfig struct {
 	// Endpoint is the Encore API endpoint that should be called when the cron job executes.
 	// It must not take any parameters other than context.Conetxt; that is, its signature must be
 	// either "func(context.Context) error" or "func(context.Context) (T, error)" for any type T.
-	Endpoint interface{}
+	Endpoint any
```

Comment:
Now that we use `any` and generics we also need to update `go.mod` to specify `go 1.18`